src/ldp.rs

```diff
@@ -615,6 +615,7 @@ async fn sign(
         created: Some(options.created.unwrap_or_else(now_ms)),
         domain: options.domain.clone(),
         challenge: options.challenge.clone(),
+        property_set: options.property_set.clone(),
```

Comment:
As mentioned in https://github.com/spruceid/ssi/pull/189#discussion_r659966782, there are other places in `ldp.rs` to update for passing properties from options into the proof. The current change updates `sign` which is a helper function used by several proof types, but there are some proof types that don't use it. Also the `prepare` functions need to be similarly updated. Basically, each place in the file where the string `Proof {` appears.

Reply:
Fixed in 538c921. However, passing `property_set` from options into proofs is still problematic, as described in https://github.com/spruceid/ssi/pull/189#discussion_r661632168:
> There may be issue options that we don't support or know about but that are supposed to have a meaning other than passing them through into the proof. e.g. credentialStatus issue option is supposed to cause a credentialStatus object to be added to the document rather than to the proof: